fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/ModdableRegistry.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.registry.sync;
+
+public interface ModdableRegistry {
```

Comment:
Maybe this should be `ModifiableRegistry`?

Reply:
Possibly, Modifiable might mean its changeable, where as this is directally a reigstry that can have modded items in.

I might remove this instance and move it to be part of the attributes.